.github/workflows/test_dbt_source_freshness.yaml

```diff
@@ -27,3 +27,22 @@ jobs:
         run: dbt source freshness --target "$TARGET"
         working-directory: ${{ env.PROJECT_DIR }}
         shell: bash
+
+      - name: Get current time
+        if: failure()
```

Comment:
In theory it might make sense to also gate this step to only run when `github.event_name == "schedule"` as in the `Send failure notification` step below, but I went with the bare `failure()` condition so that this SNS behavior matches our other workflows, e.g. `test_dbt_models`:

https://github.com/ccao-data/data-architecture/blob/cb828dfc6d8ba6178b8e20192bd2fccfc30be3c3/.github/workflows/test_dbt_models.yaml#L30-L33

It also has the benefit of simplicity, in that if we want to change the conditions under which SNS notifications are sent, we only have to edit one line (as far as I can tell there's no way to wrap conditions like these in abstractions that could be shared between steps). In any case, I'm happy to be more specific here and adjust the other workflows to match if you feel strongly about changing the pattern.

Reply:
Eh, I think this is totally fine.